shell/src/main/java/alluxio/cli/GetConf.java

```diff
@@ -114,10 +115,14 @@ public static int getConf(String... args) {
         } else {
           if (cmd.hasOption(UNIT_OPTION_NAME)) {
             String arg = cmd.getOptionValue(UNIT_OPTION_NAME).toUpperCase();
-            Unit unit;
             try {
-              unit = Unit.valueOf(arg);
-              System.out.println(Configuration.getBytes(key) / unit.getValue());
+              if (arg.endsWith("B")) {
```

Comment:
is it better to rename the current `enum` `Unit` to be `ByteUnit` in this file, and then introduce another `enum` called `TimeUnit` with possible values `MS`, `S`, `M`, `H`, `D`, just like `ByteUnit.PB`, `ByteUnit.B` and etc.
Then in here, you can do the following to be more modular? (in case we introduce other unit ending with B later on)?

```java
try {
  unit = ByteUnit.valueOf(arg);
  System.out.println(Configuration.getBytes(key) / unit.getValue());
  break;
} catch (Exception e) {
  // try next unit parse
}
try {
  unit = TimeUnit.valueOf(arg);
  System.out.println(Configuration.getTime(key, arg));
  break;
} catch (Exception e) {
  // try next unit parse
}
printHelp(String.format("%s is not a valid unit", arg));
return 1;
```